apps/web/app/(navigation)/LanguageSelector.tsx

```diff
@@ -0,0 +1,60 @@
+import { languages } from "lib/contexts";
+import * as Select from "@radix-ui/react-select";
+
+import { Flag } from "ui";
+import {
+  SelectContentStyled,
+  SelectItemStyled,
+  SelectTriggerStyled,
+} from "./LanguageSelectorStyled";
+import { useEffect } from "react";
+import { languageAtom } from "app/store";
+import { useSetAtom, useAtomValue } from "jotai";
+import { useHasScrollBar } from "lib/hooks/useHasScrollBar";
+
+export const LanguageSelector = () => {
+  const { hasScrollbar } = useHasScrollBar();
+  const setLanguage = useSetAtom(languageAtom);
+  const language = useAtomValue(languageAtom);
+
+  useEffect(() => {
+    const lang = localStorage.getItem("lang");
+    lang && setLanguage((localStorage.getItem("lang") as languages) || "en");
```

Comment:
use the const :) 
```suggestion
    const lang = localStorage.getItem("lang");
    lang && setLanguage((lang as languages) || "en");
```
I would change the second line to use if statement. IMO it is more readable. Construction with `&&` is usually used if you want to assign the result to a variable. Also, if you already check if lang is true, then you don't need the `||`

```suggestion
    const lang = localStorage.getItem("lang");
    if (lang) {
        setLanguage(lang  as languages);
   }
```

Reply:
How about: `setLanguage(lang ? lang as languages : "en")`

Reply:
done